uber/templates/preregistration/form.html

```diff
@@ -35,7 +35,7 @@
         </div>
     </div>
 
-    {% if c.DEALER_REG_START and c.DEALER_REG_PUBLIC %}
+    {% if c.DEALER_REG_START and c.DEALER_REG_PUBLIC and c.BEFORE_DEALER_REG_SHUTDOWN %}
```

Comment:
This is a separate fix - it occurred to me that we shouldn't show this link after dealer reg is shut down, since it's a bit cruel to direct people to a page telling them that they can't do what the link tells them they can do.